[FIX] website_sale_digital: `invoice_id` doesn't exists anymore

tbe-odoo opened the pull request:
- Since commit https://github.com/odoo/odoo/commit/beaa30a3d1843de43a45f419bfbc1bfa7613a920
  the object `account.invoice` has been removed.

  We now refer to `account.move` when talking about invoices.
  In website_sale_digital, a piece of code was still refering to the
  field `invoice_id` in `account.move.line` instead of using `move_id`.


--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

smetl commented on addons/website_sale_digital/controllers/main.py:
```diff
@@ -34,7 +34,7 @@ def portal_order_page(self, order_id=None, **post):
             return response
         order = response.qcontext['sale_order']
-        invoiced_lines = request.env['account.move.line'].sudo().search([('invoice_id', 'in', order.invoice_ids.ids), ('invoice_id.state', '=', 'paid')])
+        invoiced_lines = request.env['account.move.line'].sudo().search([('move_id', 'in', order.invoice_ids.ids), ('move_id.state', '=', 'paid')])
```
`'move_id.invoice_payment_state', '=', 'paid'` as `paid` is no longer a valid key inside `state`

tbe-odoo replied:
Thanks :)

I have updated the code :rocket:

smetl commented:
@tbe-odoo Are you sure you don't want to target saas-12.4? Otherwise it's ok.

tbe-odoo commented:
> @tbe-odoo Are you sure you don't want to target saas-12.4? Otherwise it's ok.

Sure, I have modified the target branch to `saas-12.4` and I have added some modifications to missing adaptations.
Could you take a last look please ? :) 

Thanks!

smetl commented:
@tbe-odoo I think you need to rebase your branch :/

smetl commented:
@robodoo r+

robodoo commented:
You may want to rebuild or fix this PR as it has failed CI.

smetl commented:
@robodoo r+

robodoo commented:
Staging failed: ci/runbot on 430a51f8d649cb5bfc5785743758d904c184f597 (view more at http://runbot.odoo.com/runbot/build/588814)

KangOl commented:
@robodoo retry

robodoo commented:
Merged at 0f6740c80c97c18d2ca8e9421c11398bf4314849, thanks!
